doc/playground.lua

```diff
@@ -0,0 +1,165 @@
+#!/usr/bin/tarantool
+
+-- How to run:
+--
+-- $ ./doc/playground.lua
```

Comment:
That should be enough:
```suggestion
-- $ doc/playground.lua
```

Reply:
Hm. It would not work if it would be just `playground.lua` in the current directory (without `.` in `PATH`, which is unusual). Why `doc/playground.lua` works? It is a bit surprising for me.

Reply:
```./``` means 'search the file in current directory (not in PATH)', not 'execute'.  Try to use absolute path with ```./``` instead of relative path.

I think this is due to the history of shell development. This is not critical and you can not change the line.

Reply:
I would prefer to keep it. I feel myself a bit more safe with the more explicit syntax.